pkg/test/integration/server.go

```diff
@@ -82,6 +86,14 @@ func NewCRDCleaner() OperationFn {
 	}
 }
 
+// NewRemoteDirCleaner cleans up the default directory where remote CRDs were
+// downloaded.
+func NewRemoteDirCleaner() OperationFn {
+	return func(e *envtest.Environment, c client.Client) error {
+		return os.RemoveAll(remotePath)
```

Comment:
Should we assume we'll always be dealing with a real filesystem here, or would there be value in using something like https://github.com/spf13/afero to let us abstract that away?

Reply:
I really like `afero`, but as long as we are using `envtest` to install our CRDs (not entirely certain we will want to forever) we will be reliant on the fact that it uses the actual real filesystem.

Reply:
Got it, fair enough.